admin-ui/src/components/settings/settings-section-card.tsx

```
@@ -1,49 +1,42 @@
-import type { ReactNode } from "react"
+import type { CSSProperties, ReactNode } from "react"
 import { forwardRef } from "react"
```

ESLint

[error] Missed spacing between "react" and "react".

(perfectionist/sort-imports)